pegen/pegen.c

```diff
@@ -767,15 +756,13 @@ seq_count_dots(asdl_seq *seq)
     int number_of_dots = 0;
     for (int i = 0, l = asdl_seq_LEN(seq); i < l; i++) {
         Token *current_expr = asdl_seq_GET(seq, i);
+        assert(current_expr->type == ELLIPSIS || current_expr->type == DOT);
         if (current_expr->type == ELLIPSIS) {
             number_of_dots += 3;
         }
         else if (current_expr->type == DOT) {
```

Comment:
If we remove the `else` then the `if` is not needed in this line. We could also transform this into a switch for clarity (and maybe some speed).